dbignite/data_model.py

```diff
@@ -33,9 +33,69 @@ def listDatabases(self) -> Iterable[Database]:
     def update(self) -> None:
         ...
 
-class FhirBundles(DataModel):
-    def __init__(self, path: str):
-        self.path = path
+class FhirBundles():
```

Comment:
we are not passing DataModel?

Reply:
Does DataModel introduce needed uniformity across classes? 

1.  _summary()_ I agree this makes sense to be defined and included as part of a FhirBundles representation
2. _update()_  I would discourage because it introduces mutability 
3. _listDatabases()_  I don't think is needed in this context

Can add DataModel back in but do not foresee it serving much purpose as a parent class.